data/pathfinder/paizo/player_companion/people_of_the_north/potn_abilities.lst

```diff
@@ -7,33 +7,46 @@ SOURCELONG:People of the North	SOURCESHORT:POTN	SOURCEWEB:http://paizo.com/produ
 # New Traits
```

Comment:
I made those traits that required it ethnicity-specific by adding PRETEMPLATE to them, and I added the snowcaster ethnicity "ability" as well as some human ethnicity "abilities" so they can get the proper templates.